pkg/util/nsenter/exec/exec.go

```diff
@@ -0,0 +1,75 @@
+// +build linux
```

Comment:
I'd put it directly into `nsenter/exec.go`, it's quite small package and it won't conflict with anything.